_health-care/_js/_components/full_name.jsx

```diff
@@ -0,0 +1,85 @@
+import React from 'react';
+import _ from 'lodash';
+
+class FullName extends React.Component {
+  constructor() {
+    super();
+    this.state = {hasError: false};
+    this.handleChange = this.handleChange.bind(this);
+  }
+
+  componentWillMount() {
+    this.id = _.uniqueId();
+  }
+
+  handleChange() {
+    const name = {
+      first: this.refs.first.value,
+      middle: this.refs.middle.value,
+      last: this.refs.last.value,
+      suffix: this.refs.suffix.value
+    }
+
+    const requiredFields = [this.refs.first, this.refs.last];
+
+    for (var i=0; i<requiredFields.length; i++) {
+      let errorDiv = requiredFields[i].parentElement;
+      
+      errorDiv.classList.remove('usa-input-error');
+
+      if (!this.validate(requiredFields[i])) {
+        this.setState({hasError: true});
+        errorDiv.classList.add('usa-input-error');
+      } else {
+        this.setState({hasError: false});
+      }
+    }
+
+    this.props.onUserInput(name);
+  }
+
+  validate(field) {
+    let valid = field !== '' && /^[a-zA-Z '\-]+$/.test(field.value);
```

Comment:
Just return the expression. 
`return field !== '' && /^[a-zA-Z '\-]+$/.test(field.value);`